### ISSUE TITLE
`cor_graphical` errors 

### ISSUE THREAD

**Issue by benkeser** [CoVPN/correlates_reporting#307]
Currently Moderna is seeing errors in `CoR_wrcdf_with_VE_lines.R` in the function call `covid_corr_rcdf_ve_lines.R`.

The error something like `Aesthetics must be of length 1 or same as the data (1):`; see [here](https://community.rstudio.com/t/error-aesthetics-must-be-either-length-1-or-the-same-as-the-data-2-fill/15579) for similar error.

Could this be related to #306 and due to the fact that there are no adjudicated cases for Baseline Seropositives?

**Comment by KenLi93** [comment]
Hi David, thank you for letting me know of the bug! I'm looking into it right now and will let you know as soon as a fix is ready

**Comment by KenLi93** [comment]
Hi David, I pushed a fix of the issue to #302. The VE curves were not plotted for baseline seropositive subjects if no cases were in that group, since the VE cannot be estimated

**Comment by benkeser** [comment]
Thanks, Kendrick.

Another bug we tracked down today:
In some cases, if the CI on the antibody scale in these plots is constant, `ggplot` will throw an error in your call to `geom_ribbon` because `dat_area_V` is empty.

The workaround I had was to add the following check [here](https://github.com/CoVPN/correlates_reporting/blob/fe6e52f13235850b399c3c5811b59f7a7b09d86d/cor_graphical/code/cor_wrcdf_plot_function.R#L46)

```r
 if(abs(line_low - line_top) < 1e-3){ 
  line_low <- line_low + 1.5e-3
  line_top <- line_top - 1.5e-3
}
```

This ensures that at least some observations end up in `dat_area_V`. We might also consider decreasing the sequence of `x`  values to be spaced closer together than `1e-3`, so that this bar isn't too wide.

I'm not sure if this is the most principled approach. Perhaps, we could make this change in order to make the plot code work, but in the legend, we could note that the CI was actually 0 width?
